downstream/modules/builder/con-gathering-system-dependencies.adoc

```diff
@@ -0,0 +1,19 @@
+[id="con-gathering-system-dependencies"]
+
+= Gathering system-level dependencies
+
+The `bindep` format provides a way of specifying cross-platform requirements. A minimum expectation is that the collection(s) specify necessary requirements for `[platform:rpm]`.
+
+Below is an example of content from a valid `bindep.txt` file:
+
+.A `bindep.txt` file
+====
+----
+python38-devel [platform:rpm compile]
+subversion [platform:rpm]
+subversion [platform:dpkg]
```

Comment:
Please remove all of the lines containing `dpkg`. We do not support debian based EEs and this will cause confusion.